pkg/heroku/heroku.go

```diff
@@ -96,7 +116,7 @@ func (c *Client) Delete(path string) error {
 //   nil         no body
 //   io.Reader   body is sent verbatim
 //   else        body is encoded as application/json
-func (c *Client) NewRequest(method, path string, body interface{}) (*http.Request, error) {
+func (c *Client) NewRequest(method, path string, body interface{}, headers http.Header) (*http.Request, error) {
```

Comment:
IMO, it might be a little cleaner to just do this:

```go
req, err := NewRequest("GET", "/", nil)
if err != nil {
        return err
}
req.Header.Set(..)
```